classes/class-wc-qd-invoice-manager.php

```diff
@@ -34,6 +34,8 @@ public function create_invoice( $order_id ) {
 			'processor' => 'woocommerce',
 			'processor_id' => $order_id,
 			'payment_method' => $this->get_payment_method($order_id),
+      			'payment_processor' => get_post_meta( $order_id, '_payment_method', true ),
```

Comment:
I was checking the WC API docs and we should use the methods `get_payment_method` and `get_transaction_id` to ensure future compatibility.

Reply:
:+1: